.circleci/config.yml

```diff
@@ -0,0 +1,34 @@
+version: 2 # use CircleCI 2.0
```

Comment:
Good idea.  Add CircleCI, so that you have CI/CD pipelines, and run tests automatically.  This will decrease technical debt.